### DISCUSSION TITLE
Documentation for framer-motion 6.x?

### DISCUSSION THREAD

**Discussion by hammad-ilyas-333** [motiondivision/motion#2389] [Q&A]
I am using react 17 and cannot migrate to framer-motion 10 right now. Is there any place where I can get the documentation of older versions?

**Comment by agrohs** [comment]
In a similar situation here and very frustrating to not be able to access the older docs.

**Comment by hyperflux** [comment]
I, too, am working on a legacy project that is stuck using React 17 and I had the same need for old Motion documentation. I was able to get exactly what I needed using the good ol' Wayback Machine! 

Here's a link to the Motion API documentation from September 2020, for example: 
https://web.archive.org/web/20200921160806/https://www.framer.com/api/motion/

**Reply by ktrip96** [reply]
Do you know what framer-motion version is this?

**Reply by secondfolder** [reply]
I'm sure this is no longer relevant for you but in case anyone else who comes across this, it seems like version 6 was the last version that officially supported react 18 (`npm install framer-motion@6.5.1`). This is the last archived docs I could find before version 7 dropped support for react 17: https://web.archive.org/web/20220706065309/https://www.framer.com/docs/

**Comment by ktrip96** [comment]
Same problem here
